src/device-registry/config/global/db-projections.js

```
@@ -5,1091 +5,1191 @@ const isEmpty = require("is-empty");
+          admin_level: 1,
+          grid_codes: 1,
+          centers: 1,
+          shape: 1,
+          createdAt: 1,
+          network: 1,
+          groups: 1,
+          sites: "$sites",
+          numberOfSites: {
+            $cond: {
+              if: { $isArray: "$sites" },
+              then: { $size: { $ifNull: ["$sites._id", []] } },
```

Biome (1.9.4)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)